torchrl/envs/libs/vmas.py

```diff
@@ -411,22 +590,18 @@ def read_info(self, infos: Dict[str, torch.Tensor]) -> torch.Tensor:
 
         return infos
 
-    def read_done(self, done):
+    def _read_done(self, done):
         done = _selective_unsqueeze(done, batch_size=self.batch_size)
         return done
 
-    def read_reward(self, rewards):
+    def _read_reward(self, rewards):
         rewards = _selective_unsqueeze(rewards, batch_size=self.batch_size)
         return rewards
 
-    def read_action(self, action):
+    def _read_action(self, action, group: str):
```

Comment:
I think this solution is worse, it used to be public and now it's private. Is there a solution that does not break previous behaviour?

Reply:
yes but it will leave the method public.

Just so to understand, do you want these methods public or private? I am happy to leave them public and making thewm bc comptible, but I thoght that private is what you wanted

Reply:
The least bc-breaking the better.
I wasn't pointing at these methods in particular, more at all the attributes created when creating specs. It seems like there are loads of them.
`read_action` is public in other wrappers so I don't think it's a problem if it's public here too.